api/src/main/kotlin/edu/wgu/osmt/db/PublishStatus.kt

```diff
@@ -1,18 +1,26 @@
 package edu.wgu.osmt.db
 
+import com.fasterxml.jackson.annotation.JsonProperty
+
 const val UNARCHIVED = "unarchived"
 const val DELETED = "deleted"
-
+const val WORKSPACE = "workspace"
 const val PUBLISHED = "published"
 const val ARCHIVED = "archived"
 const val DRAFT = "draft"
 
 enum class PublishStatus(val apiValue: String) {
+    @JsonProperty("unarchived")
```

Comment:
Please be on the lookout for this new @JsonProperty changing the API contract in unexpected ways. This is better than not declaring it, but this may pop up in unexpected places, and our test coverage over JSON payloads is pretty lean.